src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/BooleanExpressionComplexityCheck.java

```diff
@@ -74,6 +74,22 @@
  * <pre>
  * &lt;module name="BooleanExpressionComplexity"/&gt;
  * </pre>
+ * <p>Code Example:</p>
+ * <div class="wrapper">
+ * <pre>
+ * public class Test
+ * {
+ *  public static void main(String args[])
+ *  {
+ *    boolean a = true;
+ *    boolean b = false;
+ *
+ *    boolean c = (a &amp; b) | (b &amp; a); //OK
+ *    boolean d = (!a &amp;&amp; b) ^ (a || !b) &amp;&amp; a; //violation
```

Comment:
Please change the comment to `// violation: complexity is 4` for clarity. 
Here and below.

Reply:
Done